src/EventListener/RegistrationListener.php

```diff
@@ -13,10 +13,7 @@
  */
 class RegistrationListener implements EventSubscriberInterface
 {
-    /**
-     * @var TokenGenerator
-     */
-    private $tokenGenerator;
+    private TokenGenerator $tokenGenerator;
```

Comment:
This should rather use the TokenGeneratorInterface for consistency with other classes

Reply:
`TokenGenerator` is the type used by the constructor. Should the constructor be changed?

Reply:
I would say yes. Typehinting the interface is better

Reply:
Done!